helm/factset-reader/templates/pvc.yaml

```diff
@@ -8,3 +8,6 @@ spec:
   resources:
```

Comment:
Please parameterize the claim name in the deployment

Reply:
I think it is enough how it is now, as using the service-name+"pvc". In this case if we have many services the won't collide.